build1.sh

```diff
@@ -38,6 +38,9 @@ chmod +x meilix-generator.sh
 echo "$(<meilix-generator.sh)" #to test the file
 ./meilix-generator.sh
 rm meilix-generator.sh
+
+#change host name
```

Comment:
No, it does not solves #104 
I have not tested it for user name change so not including last two lines